cypress/integration/scheduled-events-timeline.spec.ts

```diff
@@ -288,6 +288,10 @@ context('Scheduled events timeline tests', () => {
 
     it('should not reset dates if page reloads', () => {
       cy.initializeSession('UserOfficer');
+      cy.visit({
+        url: '/calendar',
+        timeout: 15000,
```

Comment:
Why is the timeout here? A comment would be nice if there is a reason for it.

Reply:
That is timeout for the visit request. It is just extending the time that we wait for request to be resolved it is not waiting 15 seconds before firing the request.